server/config/config.go

```diff
@@ -156,7 +156,6 @@ func NewConfig() *Config {
 
 	fs.StringVar(&cfg.Log.Level, "L", "", "log level: debug, info, warn, error, fatal (default 'info')")
 	fs.StringVar(&cfg.Log.File.Filename, "log-file", "", "log file path")
-	fs.BoolVar(&cfg.Log.File.LogRotate, "log-rotate", true, "rotate log")
```

Comment:
not supported now?

Reply:
This config is useless. See https://github.com/pingcap/tidb/pull/12594.

Reply:
@lonng , Remove the related code about this. such as config.toml.